### PR TITLE
Fix bag create simple res tful servise#1

### PR DESCRIPTION
**Pull request by WadoZzz**
1. Added ResponseEntity
2. Added  PersonService
3. ReWritten PersonController

### CODE REVIEW

File: src/main/java/com/rest/wadoz/RESTful_Web_Service/service/PersonServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    @Override
+    public void create(Person person) {
+        personRepository.save(person);
+    }
+
+    @Override
+    public void delete(Long id) {
+        if (personRepository.findById(id).isPresent()) {
```
Comment (White28) [inline src/main/java/com/rest/wadoz/RESTful_Web_Service/service/PersonServiceImpl.java]: You don't need to find the person by id before deleting, you can just check if the person exists `personRepository.existsById(id)`

File: src/main/java/com/rest/wadoz/RESTful_Web_Service/service/PersonServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            personRepository.deleteById(id);
+        } else throw new NotFoundPersonException();
+    }
+
+    @Override
+    public void update(Long id, Person person) {
+        if (personRepository.findById(id).isPresent()){
```
Comment (White28) [inline src/main/java/com/rest/wadoz/RESTful_Web_Service/service/PersonServiceImpl.java]: The same here

File: src/main/java/com/rest/wadoz/RESTful_Web_Service/service/PersonServiceImpl.java (hunk 1 above)
Comment (WadoZzz) [reply src/main/java/com/rest/wadoz/RESTful_Web_Service/service/PersonServiceImpl.java]: fixed

File: src/main/java/com/rest/wadoz/RESTful_Web_Service/service/PersonServiceImpl.java (hunk 2 above)
Comment (WadoZzz) [reply src/main/java/com/rest/wadoz/RESTful_Web_Service/service/PersonServiceImpl.java]: fixed

**Pull request merged**